README.md

```diff
@@ -153,6 +153,8 @@ Given the objective of making this project available to creative artists, the co
 - https://www.geeksforgeeks.org/vlc-module-in-python-an-introduction/
 - [forum thread: Rpi4: YouTube Full HD 1080p is slow! How to fix?](https://forums.raspberrypi.com/viewtopic.php?t=302787)
 
+Also don't forget to checkout our [wiki](https://github.com/omarcostahamido/PVM/wiki)! It contains instructions on various topics like [setting up remote access to the Raspberry Pis](https://github.com/omarcostahamido/PVM/wiki/How-to-connect-to-a-Raspberry-Pi-remotely) and how to run the examples.
```

Comment:
- Though it is ok to have a general call for attention to the wiki at the bottom, this doesn't look like the place to refer to specific entries like it is right now.
- please move `setting up remote access to the Raspberry Pis` reference up to where it makes sense from the perspective of someone who's following along this step-by-step. I would say somewhere between `### Raspberry Pi OS` and `### Set up environment`
- it now also feels that `### Raspberry Pi OS` shall have a quick note at the end mentioning that the first time you boot up the rpi device after loading the flashed sd card it probably needs to be connected to a screen and keyboard and mouse to setup the user and password, or no? Can't remember this step.
- since you're working on readme updates please delete the empty sub-section `### Prerequisites` line 19
- please delete also line 145 which is a duplicate (my mistake)

Reply:
> please move setting up remote access to the Raspberry Pis reference up to where it makes sense from the perspective of someone who's following along this step-by-step. I would say somewhere between ### Raspberry Pi OS and ### Set up environment

Agree, that's the better place.

> it now also feels that ### Raspberry Pi OS shall have a quick note at the end mentioning that the first time you boot up the rpi device after loading the flashed sd card it probably needs to be connected to a screen and keyboard and mouse to setup the user and password, or no? Can't remember this step.

Yes, people do need to connect rpi to a screen and keyboard and mouse for at least once. I just realized that https://www.raspberrypi.com/ also has very good guides on installing the OS and first boot configuration. I'll add the link for reference.

Reply:
Done. Please have a look when you have time 😃 @omarcostahamido. I also added a `.gitattributes` file to make the line endings consistent when the repository is receiving commits from collaborators from different operating systems (e.g. Windows and Mac).

Reply:
@rustberry  I'm going to try to add a commit to the branch with some re-articulations. Easier than to just try to explain here. Thank you